src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -202,19 +207,16 @@ protected boolean mustCheckName(DetailAST ast) {
      * @return the access modifier of the method/constructor.
      */
     private static AccessModifier getAccessModifier(final DetailAST ast) {
-        final DetailAST params = ast.getParent();
```

Comment:
Check is changed in non simple way - we need do regression diff testing, please run on all projects.
There will be too much differences if we run them in simple way as we do usually.
So we need to make master config with only one Check ParameterNameCheck , Patch config with two Checks (ParameterNameCheck, LambdaParameterName) to make sure that after change violations are stay as before, the only difference is name of Check in violation.

Reply:
Done. Actually there no violations on projects except of Vavr.
https://kazachka.github.io/parameter-name-check-diff/